### DISCUSSION TITLE
有2个网口无法上网，显示port3 entered blocking state

### DISCUSSION THREAD

**Discussion by fucao** [immortalwrt/immortalwrt#499] [General]
在esxi上安装的immortalwrt，有4个网口，但有eth1和eth2网口无法正常上网，这2个网口都是做个直通的。eht3上wan口，网线连接eth0是可以正常上网的。请问是我哪里设置的有问题吗？
![image](https://user-images.githubusercontent.com/15061002/138593559-c024e113-c32a-4d90-a385-2cc810d8561e.png)
![image](https://user-images.githubusercontent.com/15061002/138593587-29d99924-948b-4291-b1cb-728faacd6a1b.png)
![image](https://user-images.githubusercontent.com/15061002/138593630-a9f2d2c9-67f1-40cf-9fac-cd1c9bcae778.png)

**Comment by fucao** [comment]
好奇怪 为啥关掉呢，也没给个解答啥的

**Reply by 1715173329** [reply]
As it was moved here.

**Comment by 1715173329** [comment]
建议是不使用直通。
